src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40 * days;
+  let amount = 0;
+
+  if (days < 3) {
+    amount = rent;
+  }
+
+  if (days >= 3 && days < 7) {
+    amount = rent - 20;
+  }
+
+  if (days >= 7) {
+    amount = rent - 50;
+  }
```

Comment:
what a magic numbers 🪄
lets create variables and then use them
it is better for managing them from one place and for understanding what do they mean